packages/core/src/constant.ts

```diff
@@ -32,10 +32,14 @@ export const SOLID_CLIENT_AUTHN_KEY_PREFIX = "solidClientAuthn:";
 export const PREFERRED_SIGNING_ALG = ["ES256", "RS256"];
 
 export const EVENTS = {
+  // Note that an `error` events MUST be listened to: https://nodejs.org/dist/latest-v16.x/docs/api/events.html#error-events.
+  ERROR: "error",
```

Comment:
Where we emit error events, you can also use `emitter.listenerCount("error")` before actually emitting